### PR TITLE
Data format

### LINKED ISSUE #54: Does model support channels_first data_format?

### PR DESCRIPTION
**Pull request by fsx950223**
Add channel first support.
Fix #54.

### CODE REVIEW

**Comment by fsx950223** [comment]
It's 0.0794ms on GPU with batch_size=1, FPS: 12.594458438287154
It's 0.0462ms on GPU with batch_size=32, FPS: 21.62466738705676
GTX 1080TI, NCHW.

File: efficientdet/inference.py [hunk 1]
```diff
@@ -337,7 +337,7 @@ def __init__(self,
 
     self.signitures = None
     self.sess = None
-    self.disable_pyfun = True
+    self.disable_pyfun = False
```
Comment (mingxingtan) [inline efficientdet/inference.py:340]: Is it intended to set disable_pyfun as False? Isn't tf version better?

File: efficientdet/inference.py [hunk 2]
```diff
@@ -520,7 +522,7 @@ def __init__(self,
       self.params.update(dict(image_size=image_size))
     if num_classes:
       self.params.update(dict(num_classes=num_classes))
-    self.disable_pyfun = False
+    self.disable_pyfun = True
```
Comment (mingxingtan) [inline efficientdet/inference.py:525]: Also here.

**Comment by fsx950223** [comment]
The improvement may come from the VOC dataset rather than NCHW. Since top_k's performance highly depends on num_classes.

**Pull request merged**